samples/Directory.Build.props

```diff
@@ -3,7 +3,7 @@
   <Import Project="../eng/Compilers.props" />
   <Import Project="../eng/Versions.props" />
   <PropertyGroup>
-    <NoWarn>$(NoWarn);CS8321</NoWarn>
+    <NoWarn>$(NoWarn);CS8321;CS1591</NoWarn>
```

Comment:
It would be nice if in here we add the meaning of the warnings for future reference.

```suggestion
    <!--
         CS8321: The local function is not declared
         CS1591: Missing XML comment for publicly visible type or member
    -->
    <NoWarn>$(NoWarn);CS8321;CS1591</NoWarn>
```